src/EventBus.RabbitMQ.Standard/Options/RabbitMqOptions.cs

```diff
@@ -1,10 +1,13 @@
-﻿namespace EventBus.RabbitMQ.Standard.Options
+﻿using RabbitMQ.Client;
+
+namespace EventBus.RabbitMQ.Standard.Options
 {
     public class RabbitMqOptions
     {
         public string AutofacScopeName { get; set; }
         public string BrokerName { get; set; }
         public string Host { get; set; }
+        public int Port { get; set; } = AmqpTcpEndpoint.DefaultAmqpSslPort;
```

Comment:
The static port referenced here does not seem to work in runtime as in you need to set the `Port`